### DISCUSSION TITLE
Can't get wordmat running on any version

### DISCUSSION THREAD

**Discussion by RacksMcMoney** [Eduap-com/WordMat#136] [Support] [answered]
Tried installing both 1.26, 1.25.2 and 1.24.2, neither of them can start. When i press Alt+M and open the equation it opens up, but when i try to calculate it, it tries to start up and says:

Run-time error '-2147221164(80040154)':

Class not registered

is says that its problably installed wrongfully and says "try to disable antivirus"
ive tried that, no luck.

Im running windows 10 pro, only here ive had the issue. The 1.25.2 works fine on my laptop, which is running win11 
Any ideas of what i can do?

**Comment by soegaard** [comment]
Which Word version? 
Rember to update Word before installing WordMat.

**Reply by RacksMcMoney** [reply]
Jeg har lige lavet en frisk installation af Word, samt de andre Office programmer. Det er den samme version som den kører på min laptop. Tak for det hurtige svar
![Wordversion](https://github.com/Eduap-com/WordMat/assets/142725122/e4ca631c-0bbe-4f8b-8717-d532dc61653a)

**Comment by soegaard** [comment, marked as answer]
Måske kan rådene her hjælpe?

 https://github.com/Eduap-com/WordMat/discussions/67

**Reply by RacksMcMoney** [reply]
Jeg takker mange gange. Det fiksede mit problem. Jeg undskylder for min dovenskab, burde måske lige have kigget mere end første side igennem.

**Reply by soegaard** [reply]
> Jeg takker mange gange. Det fiksede mit problem.
Godt at høre

> ... burde måske lige have kigget mere end første side igennem.
Det var nu heller ikke helt nemt at vide, at den side fandtes.
